include/seqan/seq_io/fasta_fastq.h

```diff
@@ -375,7 +375,10 @@ inline void readRecord(TIdString & meta, TSeqString & seq, TFwdIterator & iter,
     {
         skipUntil(iter, qualCountDown);                     // skip Fastq qualities
     }
-    skipUntil(iter, TFastqBegin());                         // forward to the next '@'
+    skipUntil(iter, NotFunctor<IsWhitespace>());        // skip Fastq qualities
+    TFastqBegin fastqBegin;
+    if(!fastqBegin(*(iter)) && *(iter) != '\xff' )  // check if next record follows
+        throw ParseError("quality line(s) have different length");
 }
```

Comment:
Description is not very clear, maybe instead:
"Fastq quality string expected to be of same length as sequence string, but was not."